site/app/templates/forum/ShowCategories.twig

```diff
@@ -9,127 +9,137 @@
+						<span class="categorylistitem-desc">
+							<span class="categorylistitem-desc-text" style="word-wrap: break-word;">{{ category.category_desc }}</span>
+							<a class="post_button" title="Edit Category Description" aria-label="Edit Category Description">
+								<i class="fas fa-edit category-button edit-category-desc-button"></i>
+							</a>
+						</span>
+						<span class="categorylistitem-editdesc" style="display: none;">
+							<input type="text" aria-label="New Description of Category" placeholder="New Description of Category" class="edit-category-desc-input" style="padding: 0;" maxlength="50" onkeyup="checkInputMaxLength($(this))">
+							<a class="post_button" title="Save Changes" aria-label="Save Changes">
+								<i class="fas fa-check category-button save-desc-button"></i>
+							</a>
+							<a class="post_button" title="Cancel Changes" aria-label="Cancel Changes">
+								<i class="fas fa-times category-button cancel-desc-button"></i>
+							</a>
+						</span>
+						<div style="float: right;width: auto;">
+							<select aria-label="Category Color Picker" class='category-color-picker' style="color: white;font-size: 14px;height: 18px;padding: 0px;">
+								{% for color_name, color_code in category_colors %}
+									<option value="{{ color_code }}" style="color: white;background-color: {{ color_code }};" {% if color_code == category.color %} selected="selected" {% endif %}>{{ color_name }}</option>
+								{% endfor %}
+							</select>
+							&nbsp;
+							<a class="post_button" title="Delete Category" aria-label="Delete Category">
+								<i class="fas fa-trash category-button delete-category-button"></i>
+							</a>
+						</div>
+					</li>
+				{% endfor %}
+			</ul>
+
+		</div>
+	</div>
 </div>
 
 <script>
-    $(function() {
-        $("#ui-category-list").sortable({
-            items : '.category-sortable',
-            handle: ".handle",
-            update: function (event, ui) {
-                reorderCategories('{{ csrf_token }}');
-            }
-        });
+	$(function () {
+$("#ui-category-list").sortable({
+items: '.category-sortable',
+handle: ".handle",
+update: function (event, ui) {
+reorderCategories('{{ csrf_token }}');
+}
+});
+
+function getItem(input) {
+return $(input).closest('.category-item-box');
+}
 
-        function getItem(input) {
-            return $(input).closest('.category-item-box');
-        }
-        
-        $(".delete-category-button").click(function() {
-            const item = getItem(this);
-            const category_id = item.data('category_id');
-            const category_desc = item.data('category_desc');
-            deleteCategory(category_id, category_desc, '{{ csrf_token }}');
-        });
+$(".delete-category-button").click(function () {
+const item = getItem(this);
+const category_id = item.data('category_id');
+const category_desc = item.data('category_desc');
+deleteCategory(category_id, category_desc, '{{ csrf_token }}');
+});
```

Comment:
Please only change spacing/style when needed (this applies to everywhere you changed spacing/style in this PR). This format you are proposing does not follow our linter guidelines. Our linter does not run on twig files.